tiledb/sm/compressors/test/link_webp_test1.cc

```diff
@@ -0,0 +1,332 @@
+// stripped to point of non-useability to demonstrate linkage success
```

Comment:
For link-test purposes couldn't we just call `WebPMalloc`? (and potentially one other function from each installed target?)